MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include LICENSE
+
+recursive-include test *.py
+
+exclude tox.ini
```

Comment:
Let's play it safe and include `tox.ini`. It will keep us consistent and avoid breakage if we ever move away from something unsupported by `python setup.py test` yet supported by `tox`